flake.nix

```diff
@@ -6,7 +6,7 @@
 
     # Backend inputs
     shared-kernel = {
-      url = "github:nammayatri/shared-kernel";
+      url = "path:/Users/chakradhar.k/Projects/shared-kernel";
```

Comment:
![something-bad-is-going-to-happen-anthony](https://github.com/nammayatri/nammayatri/assets/54761970/58a2f0ff-2ab6-4362-a12f-d1332e4e08f4)